### PR TITLE
UnmakeBundle

### PR DESCRIPTION
**Pull request by psalamontech**

### CODE REVIEW

File: src/Command/UnmakeEntityCommand.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        $qb->select('a')->from($entitySelected, 'a');
+
+        $filterCounter = 0;
+        do {
+            $filterCounter = $this->addFilter($input, $output, $qb, $filterCounter);
+        } while ($this->askForAnotherProperty($input, $output));
+
+
+        $result = $qb->getQuery()->getResult();
+        $output->writeln('<info>Number of rows affected:</info> ' . count($result));
```
Comment (ivan-ivakic) [inline src/Command/UnmakeEntityCommand.php]: There is a helper `->info` and I would usually make use of `sprintf` to format the message without concatenation.

File: src/Command/UnmakeEntityCommand.php [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        $entityList = $this->entityManager->getMetadataFactory()->getAllMetadata();
+        return array_map(static function ($metadata) {
+            return $metadata->getName();
+        }, $entityList);
+    }
+
+    private function selectEntity(InputInterface $input, OutputInterface $output, array $entityNames): ?string
+    {
+        $helper = $this->getHelper('question');
+        $question = new Question('<question>What entity do you want to delete?</question> ');
```
Comment (ivan-ivakic) [inline src/Command/UnmakeEntityCommand.php]: Why do we need to write `<question>`?

Comment (psalamontech) [reply src/Command/UnmakeEntityCommand.php]: When we write `<question>`, it uses different color coding in console output, which is designed to make it easier to guide the user through command.

Comment (ivan-ivakic) [reply src/Command/UnmakeEntityCommand.php]: How come using `Question` does not apply that automatically? Can we investigate if there is some option that we can pass to the class or something else to make the formatting automatic?

Comment (psalamontech) [reply src/Command/UnmakeEntityCommand.php]: I improved the code by using the `SymfonyStyle` class, which automatically applies the question-style formatting. Previously, the implementation lacked the `SymfonyStyle` class, so the formatting wasn't applied automatically.

**Review by ivan-ivakic — APPROVED** [review]

**Pull request merged**